### PR TITLE
fixes #384

### LINKED ISSUE #384: Saving nodes without image on image field when cardinality is set to 1 causes an error
On Umami installation profile the article node type has image field with cardinality set to allow uploading single image. If a user doesn't upload an image to that field and tries to save a node, it causes the following error:
screen shot 2018-08-22 at 00 17 37

The field is not configured to be required so that shouldn't be the cause for the problem.

Here's a gif to help to reproduce the problem:
image_upload_empty_error

### PR DESCRIPTION
**Pull request by mattgrill**
## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/384

_fwiw_, we really need to talk about a constraints system at Drupal Europe.

## Testing instructions

- make an article, don't upload an image, actually save the node.

### CODE REVIEW

**Comment by dawehner** [comment]
> fwiw, we really need to talk about a constraints system at Drupal Europe.

Add it to our list on https://github.com/jsdrupal/drupal-admin-ui/issues/424

File: src/components/05_pages/NodeAddForm/index.js [hunk 1]
```diff
@@ -8,13 +8,35 @@ import PageTitle from '../../02_atoms/PageTitle/PageTitle';
...
+    this.props.contentAdd(
+      this.cleanupRelationships({
+        ...entity,
+        type: `${this.props.entityTypeId}--${this.props.bundle}`,
+      }),
+      this.props.bundle,
+    );
   };
 
+  cleanupRelationships = ({ relationships, ...rest }) => ({
```
Comment (dawehner) [inline src/components/05_pages/NodeAddForm/index.js]: Are you sure this is not a problem on node edit too?

**Comment by mattgrill** [comment]
@dawehner

File: src/components/05_pages/NodeAddForm/index.js (hunk 1 above)
Comment (mattgrill) [reply src/components/05_pages/NodeAddForm/index.js]: It is.

**Pull request merged**